x-pack/legacy/plugins/infra/public/pages/logs/analysis/setup/initial_configuration_step/analysis_setup_indices_form.tsx

```diff
@@ -75,14 +93,42 @@ const indicesSelectionLabel = i18n.translate('xpack.infra.analysisSetup.indicesS
   defaultMessage: 'Indices',
 });
 
-const formatValidationError = (validationError: IndicesValidationError) => {
-  switch (validationError) {
-    case 'TOO_FEW_SELECTED_INDICES':
-      return i18n.translate(
-        'xpack.infra.analysisSetup.indicesSelectionTooFewSelectedIndicesDescription',
-        {
-          defaultMessage: 'Select at least one index name.',
-        }
+function errorToI18n(error: ValidationIndicesUIError): React.ReactNode {
```

Comment:
No reason. This function was in a different file earlier and it was the name that made sense there since it converted an error to a I18n'ez string. It kept its name when I moved it back here.